crewai/CrewAI Observability.json

```diff
@@ -0,0 +1 @@
ens), `Output Tokens` = sum(output_tokens), `Total` = sum(total_tokens), by: {agent_name}","visualization":"table","visualizationSettings":{"autoSelectVisualization":true,"table":{"hideColumnsForLargeResults":true}},"querySettings":{"maxResultRecords":1000,"defaultScanLimitGbytes":500,"maxResultMegaBytes":1,"defaultSamplingRatio":10,"enableSampling":false},"davis":{"enabled":false,"davisVisualization":{"isAvailable":true}}},"20":{"type":"markdown","content":"## Token & Model Usage Statistics"},"21":{"type":"data","title":"Tokens by I/O type, system and response model","query":"timeseries { sum(gen_ai.client.token.usage), value.A = avg(gen_ai.client.token.usage, scalar: true) }, by: { gen_ai.token.type, gen_ai.system, gen_ai.response.model }","queryConfig":{"version":"18.0.1","subQueries":[{"id":"A","isEnabled":true,"datatype":"metrics","metric":{"key":"gen_ai.client.token.usage","aggregation":"sum"},"by":["gen_ai.token.type","gen_ai.system","gen_ai.response.model"],"filter":""}]},"subType":"dql-builder-metrics","visualization":"lineChart","davis":{"enabled":false,"davisVisualization":{"isAvailable":true}},"visualizationSettings":{"autoSelectVisualization":true,"table":{"hideColumnsForLargeResults":true}},"querySettings":{"maxResultRecords":1000,"defaultScanLimitGbytes":500,"maxResultMegaBytes":1,"defaultSamplingRatio":10,"enableSampling":false}},"22":{"type":"data","title":"Operation duration (avg. by model and op)","query":"timeseries { avg(gen_ai.client.operation.duration), value.A = avg(gen_ai.client.operation.duration, scalar: true) }, by: { gen_ai.system, gen_ai.response.model, gen_ai.operation.name }","queryConfig":{"version":"18.0.1","subQueries":[{"id":"A","isEnabled":true,"datatype":"metrics","metric":{"key":"gen_ai.client.operation.duration","aggregation":"avg"},"by":["gen_ai.system","gen_ai.response.model","gen_ai.operation.name"]}]},"subType":"dql-builder-metrics","visualization":"lineChart","davis":{"enabled":false,"davisVisualization":{"isAvailable":true}},"visualizationSettings":{"autoSelectVisualization":true,"table":{"hideColumnsForLargeResults":true}},"querySettings":{"maxResultRecords":1000,"defaultScanLimitGbytes":500,"maxResultMegaBytes":1,"defaultSamplingRatio":10,"enableSampling":false}}},"layouts":{"0":{"x":3,"y":0,"w":4,"h":2},"1":{"x":7,"y":0,"w":4,"h":2},"2":{"x":0,"y":21,"w":24,"h":5},"3":{"x":8,"y":35,"w":8,"h":6},"4":{"x":0,"y":35,"w":8,"h":6},"6":{"x":3,"y":2,"w":4,"h":2},"7":{"x":0,"y":28,"w":24,"h":3},"8":{"x":0,"y":8,"w":24,"h":4},"9":{"x":0,"y":18,"w":24,"h":3},"10":{"x":0,"y":31,"w":24,"h":1},"11":{"x":0,"y":0,"w":3,"h":4},"12":{"x":7,"y":2,"w":4,"h":2},"13":{"x":11,"y":2,"w":9,"h":2},"14":{"x":0,"y":26,"w":24,"h":2},"15":{"x":0,"y":16,"w":24,"h":2},"16":{"x":11,"y":0,"w":9,"h":2},"17":{"x":20,"y":0,"w":4,"h":2},"18":{"x":20,"y":2,"w":4,"h":2},"19":{"x":0,"y":32,"w":24,"h":3},"20":{"x":0,"y":4,"w":24,"h":1},"21":{"x":0,"y":5,"w":24,"h":3},"22":{"x":0,"y":12,"w":24,"h":4}},"importedWithCode":false,"settings":{}}
```

Comment:
I see `gen_ai.system` used in some DQLs of the dashboard. OpenLLMetry moved to `gen_ai.provider.name` a few versions ago to align with the OpenTelemetry SemConv. 
I suggest rewriting the queries, adding both fields if you filter or pivot on a metric